### DISCUSSION TITLE
프로젝트 주제

### DISCUSSION THREAD

**Discussion by jgjgill** [prgrms-web-devcourse/React-Playground-Study#9] [Project]
프로젝트로 진행하면 좋을 것 같은 주제에 대해 미리 이야기를 나눠봅니다!
자유롭게 올려주세요..! 🙇‍♂️

**Comment by jgjgill** [comment]
좋은 루틴을 만드는데 관심이 많습니다..!
제가 생각하기에 습관까지 형성되는데 제일 어려운 시점이 **3주**인데요..!

이에 작심삼일을 모티브로 **작심삼주**라는 프로젝트를  생각해봤습니다! 😆
3주 동안 목표한 일을 체크하여 성취했을 시 보상? 배지? 같은 것을 부여하여 성취감을 얻고자 하는 것입니다!

**Reply by backward99** [reply]
목표를 이루기 위해 도움이 될 만한 행동들을 추천 목록으로 추가 해 놓는 것도 좋을 것 같네요!

**Reply by jkea1** [reply]
좋은 루틴 만드는데 관심이 많군요!! 저도 운동을 꾸준히 하고 싶어해서 작심삼주 어플이 있으면 좋을 거 같습니다!! 사진 같은 것도 올릴 수 있는 기능이 있으면 좋을 거 같아요! 👍
